example/windows/.gitignore

```diff
@@ -0,0 +1,17 @@
+flutter/ephemeral/
```

Comment:
We don't support windows, can we remove it?

Reply:
Removed it.